adx-queries/README.md

```diff
@@ -0,0 +1,78 @@
+# ADX plugin sample queries and walkthrough
```

Comment:
ADT query plugin for ADX: Sample Queries and Walkthrough

Reply:
Please change any references of "ADX Query" as in the title to ADT Query Plugin for ADX: Sample Queries, or something similar to ensure the ADT/ADX reference.